infra_tooling_account/app.py

```diff
@@ -26,8 +26,9 @@
 
 TAGS = {"project_name": PROJECT_NAME, "stage_name": STAGE_NAME}
 
-settings_provider = LocalFileSystemSettingsProvider("../config/settings")
-settings_provider.validate_settings()
+settings = Settings.from_file_path("../config/settings")
+settings_validator.validate(settings)
```

Comment:
I guess it would depend on a user, what would be installed in the landing zone first: tooling or monitoring parts. And I believe it's better to validate settings as soon as possible. I would prefer leaving validations in both stacks just in case.

Reply:
@npaskrobka , let's discuss your idea separately from this PR (maybe, during the next sync-up). It looks like I didn't fully understand it.

Another idea worth a separate discussion - forcing user to create tooling infra first before creating corresponding monitored infra. I like your idea, but we can achieve it differently maybe :) Let's brainstorm